### PR TITLE
Implement add book and search book.

### PR DESCRIPTION
**Pull request by Vivian151198**
Please approve!!

### CODE REVIEW

File: src/ExerciseOOP/Book.java [hunk 1]
```diff
@@ -48,4 +48,14 @@ public String getAvailabilityStatus() {
     public void setAvailabilityStatus(String availabilityStatus) {
         this.availabilityStatus = availabilityStatus;
     }
+
+    @Override
```
Comment (thinhkhang97) [inline src/ExerciseOOP/Book.java:52]: Why do we have @Override annotation here?

File: src/ExerciseOOP/Library.java [hunk 2]
```diff
@@ -1,4 +1,77 @@
...
 public class Library {
+
+    public static void main(String[] args) {
+        List<Book> listBook = new ArrayList<>();
+        Book book1 = new Book("Doanh Nghiep The Ki 21", "Robert", "1/1/2000", "Available");
+        Book book2 = new Book("NDCD", "Bob Burg", "1/1/2000", "Available");
+        Book book3 = new Book("NLDKCD", "Robin Sharma", "1/1/2000", "Available");
+        listBook.add(book1);
+        listBook.add(book2);
+        listBook.add(book3);
```
Comment (thinhkhang97) [inline src/ExerciseOOP/Library.java]: I think you should move these code into a function such as `initBooks`

File: src/ExerciseOOP/Library.java [hunk 3]
```diff
@@ -1,4 +1,77 @@
...
+        System.out.println("---------MENU------------");
+        System.out.println("1. Add book");
+        System.out.println("2.Search book");
+        System.out.println("3. Borrow book");
+        System.out.println("4. Return book");
+        System.out.println("5. Display book");
+        System.out.println("6. Exit");
```
Comment (thinhkhang97) [inline src/ExerciseOOP/Library.java]: Move to function `showMenu`

File: src/ExerciseOOP/Library.java [hunk 4]
```diff
@@ -1,4 +1,77 @@
...
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("What do you want?. Please type the number as the menu");
+        int userType = scanner.nextInt();
+        if (userType == 1) {
+            listBook = addBook();
+            System.out.println(listBook.get(0).toString());
```
Comment (thinhkhang97) [inline src/ExerciseOOP/Library.java:17]: Each user function should be move to a function. For example, these code should be in a handleAddBook

**Review by thinhkhang97 — CHANGES_REQUESTED** [review]

File: src/ExerciseOOP/Book.java (hunk 1 above)
Comment (Vivian151198) [reply src/ExerciseOOP/Book.java:52]: Because I use this method to print the object

File: src/ExerciseOOP/Library.java (hunk 2 above)
Comment (Vivian151198) [reply src/ExerciseOOP/Library.java]: I got it, thanks

File: src/ExerciseOOP/Library.java (hunk 3 above)
Comment (Vivian151198) [reply src/ExerciseOOP/Library.java]: Ok

**Review by thinhkhang97 — APPROVED** [review]

**Pull request merged**